src/features/profile/pages/ChangePassword.test.tsx

```diff
@@ -105,11 +101,11 @@ describe('ChangePassword', () => {
     const continueButton = getByTestId('button-container')
     fireEvent.press(continueButton)
 
-    await waitFor(() => {
-      expect(displaySuccessSnackBar).toBeCalledWith({
-        message: 'Mot de passe modifié',
-        timeout: SNACK_BAR_TIME_OUT,
-      })
+    await act(() => flushAllPromisesTimes(15))
```

Comment:
Même superFlush ne suffit pas ? 😆 
```suggestion
    await superFlushWithAct()
```

Reply:
OK